### PR TITLE
feat(backend): add API key expiration field

### PR DESCRIPTION
**Pull request by stanleynguyen**
## Problem

[Ticket](https://www.notion.so/opengov/DB-Migration-to-add-valid_util-and-notification_emails-a9e9460aa4894386b643a6eaef7d699c?pvs=4)

## Solution

Implement it and default existing ones to be valid until 21st Apr 2024 (1 year from expected release)

## Deployment Checklist

_Any tasks that need to be done on the environment before releasing of this PR (e.g. Setting environment variables, running migrations, etc). If there's no task to be done, put "N/A"_

- [x] Migrate DB

### CODE REVIEW

**Review by richard-yak — APPROVED** [review]

File: backend/src/database/migrations/20230408151021-add-api-key-expiry-and-contact.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.addColumn('api_keys', 'valid_until', {
+      type: Sequelize.DataTypes.DATE,
+      allowNull: false,
+      defaultValue: new Date('2024-04-21T00:00:00.000Z'),
```
Comment (tan-zx) [inline backend/src/database/migrations/20230408151021-add-api-key-expiry-and-contact.js:8]: Do we want to use a default time that is more intuitive to users in Singapore?

Comment (tan-zx) [reply backend/src/database/migrations/20230408151021-add-api-key-expiry-and-contact.js:8]: E.g. 2024-04-20 11:59pm GMT+8

File: backend/package.json [hunk 2]
```diff
@@ -53,6 +53,7 @@
     "libphonenumber-js": "1.10.6",
     "lodash": "4.17.21",
     "module-alias": "2.2.2",
+    "moment": "^2.29.4",
```
Comment (tan-zx) [inline backend/package.json:56]: by convention, think we're freezing the dependencies (realise I didn't do that for `yamljs` below oops)

**Review by tan-zx — APPROVED** [review]

File: backend/src/database/migrations/20230408151021-add-api-key-expiry-and-contact.js (hunk 1 above)
Comment (tan-zx) [reply backend/src/database/migrations/20230408151021-add-api-key-expiry-and-contact.js:8]: actually, now that I look at it `21 April 2024, 8am` seems pretty intuitive to me too

image

File: backend/package.json (hunk 2 above)
Comment (stanleynguyen) [reply backend/package.json:56]: Will freeze both

Comment (tan-zx) [reply backend/package.json:56]: thanks boss

**Pull request merged**